src/contour/ui.template/main.qml.in

```diff
@@ -39,11 +39,13 @@ ApplicationWindow
     onWidthChanged : function() {
         vtui.width = width
         vtui.session.requestWindowResize(width, height)
+        vtui.updateSizeWidget()
     }
 
     onHeightChanged: function() {
         vtui.height = height
         vtui.session.requestWindowResize(width, height)
+        vtui.updateSizeWidget()
```

Comment:
same commet for onWidthChanged. Okay, this is why it's shown on a per-pixel basis. I'd recomment to actually connect/hook into the pageSize (lines/columns) change event instead. :)

Reply:
As discussed in discord
we check uptime of the contour and show resize widget only after one second